### PR TITLE
This feature is going to Update Task

### PR DESCRIPTION
**Pull request by LouangeMu**
### What does this PR do?
This Pull request is going to Update Task.

### Description of Task to be completed?

The user will be able to Update Task.

### How should this be manually tested?

- Clone this GitHub repo (this can be done either by directly downloading the repo or using your terminal and running git clone https://github.com/Soma-Technologies-Inc/Backend-To-Do-App.git
- Open your terminal
- cd into the repo folder which will be created in your local machine
- Then run npm install
- Then run node src/app.js
- Open your Postman (these are the tools used to test apis) then make a request to the endpoint provided

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
N/A
### Screenshot(If appropriate)
PIC1

### CODE REVIEW

**Comment by BenieSereine** [comment]
Everything is fine

**Comment by EngMutabazi** [comment]
good!!

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import routes from "./routes/index";
+import router from "./routes/index";
+const app=express();
+app.use(express.json());
+app.use(bodyParser.json());
+app.use('/task',router)
+const port=process.env.PORT || 3000;
+app.listen(port,()=>console.log(`server is running at ${port}`));
+export default app;
```
Comment (tigthor) [inline src/app.js]: Remove this line it has nothing to do in this file

**Review by tigthor — CHANGES_REQUESTED** [review]
Please Remove the last line in src/app.js which `export app `.

File: src/app.js (hunk 1 above)
Comment (LouangeMu) [reply src/app.js]: Done

**Review by tigthor — APPROVED** [review]
Well done

**Comment by LouangeMu** [comment]
This is the best email ever
My PR is now approved ahwiiii

**Pull request closed without merging**